bom/pom.xml

```diff
@@ -1337,6 +1337,21 @@
                 <artifactId>helidon-inject-service</artifactId>
                 <version>${helidon.version}</version>
             </dependency>
+            <dependency>
+                <groupId>io.helidon.inject</groupId>
+                <artifactId>helidon-inject-codegen</artifactId>
+                <version>${helidon.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>io.helidon.inject</groupId>
```

Comment:
maybe you have this later in the PR, but dropping this comment here at the start.

Do we detect when we have jakarta and javax on the classpath at the same time? We should, and there should be some way to establish a meta config preference on which to resolve to, and a failure to detect that flag should fail fast at compile time.

Reply:
We do not care whether you have jakarta or javax on the classpath, we support both and do not have a conflict on our side

Reply:
And also - the bom pom may be inconsistent for a while. I will fix it when I will be removing modules.